scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the statistics (mean, stddev) of scalar maps, which can represent
+diffusion metrics, in a ROI.
+
+The mask can either be a binary mask, or a weighting mask. If a weighting mask
+should either contain floats between 0 and 1, or should be normalized with
+--normalize_weights.
+
+IMPORTANT: if the mask contains weights (and not 0 and 1 exclusively), the
+standard deviation will also be weighted.
+"""
+
+import argparse
+import nibabel as nib
+import numpy as np
+import os
+import json
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_json_args,
+                             assert_inputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.utils.metrics_tools import get_roi_metrics_stats
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_mask',
+                   help='Mask volume file name, formatted in any nibabel ' +
+                        'supported format.\nCan be a binary mask or a ' +
+                        'weighted mask.')
+
+    g = p.add_mutually_exclusive_group(required=True)
+    g.add_argument('--metrics_dir',
+                   help='Metrics files directory. name of the directory ' +
+                        'containing the metrics files.')
+    g.add_argument('--metrics', dest='metrics_file_list', nargs='+',
+                   help='Metrics nifti file name. list of the names of the ' +
+                        'metrics file, in nifti format.')
+
+    p.add_argument('--bin', action='store_true',
+                   help='If set, will consider every value of the mask ' +
+                        'higher than 0 to be part of the mask, and set to 1 ' +
+                        '(equivalent weighting for every voxel).')
+
+    p.add_argument('--normalize_weights', action='store_true',
+                   help='If set, the weights will be normalized to the [0,1] '
+                        'range.')
+
+    add_overwrite_arg(p)
+    add_json_args(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.in_mask, [os.listdir(args.metrics_dir),
```

Comment:
BUG here. If user used --metrics, then args.metrics_dir is None but this (`os.listdir(None)` still gives the list of dirs in the current directory.

If user used --metrics_dir (I tried with --metrics_dir test_metrics, a folder with one FA inside), then what you send to the optional parameter is [['fa.nii.gz'], None]. So two bugs: 1- the dir name is not attached (should be test_metrics/fa.nii.gz) and 2-it's a list inside a list so it tries to see if the file ['fa.nii.gz'] exists and crashed saying it's a list, not a string.